Multisig/Cross-layer/Logger/LogService.swift

```diff
@@ -118,26 +118,21 @@ public final class LogService: Logger {
     /// See `CrashlyticsLogger` and `ConsoleLogger` for details.
     ///
     /// - Parameter bundle: Bundle to initialize the logger with. By default, it is the main bundle.
-    init(bundle: BundleProtocol = Bundle.main) {
-        let string = bundle.object(forInfoDictionaryKey: LogLevelKey) as? String ?? ""
-        level = LogLevel(string: string)
-        addLoggers(from: bundle)
+    init() {
+        level = LogLevel(string: App.configuration.app.logLevel)
+        addLoggers()
     }
 
-    private func addLoggers(from bundle: BundleProtocol) {
-        guard let enabledLoggers = bundle.object(forInfoDictionaryKey: LoggersKey) as? String else {
-            return
-        }
-        let normalizedEnabledLoggers = enabledLoggers.split(separator: ",").map {
+    private func addLoggers() {
+        let normalizedEnabledLoggers = App.configuration.app.loggers.split(separator: ",").map {
```

Comment:
App.configuration.app is used twice, it's good if we define it as private field

Reply:
I think in this case it is okay not to have it as a private field. This would just add an extra line of code :)
In our prev project, we had a very flexible approach to moving such things to a private field. In the majority of the cases, we did not do it. But we can discuss it during our learning group meeting.